metpy/calc/kinematics.py

```diff
@@ -432,3 +432,39 @@ def geostrophic_wind(heights, f, dx, dy):
     grad = _gradient(heights, *deltas)
     dy, dx = grad[-2:]  # Throw away unused gradient components
     return -norm_factor * dy, norm_factor * dx
+
+
+@exporter.export
+@check_units('[length]', '[temperature]')
+def montgomery_streamfunction(height, temperature):
+    r"""Compute the Montgomery Streamfunction on isentropic surfaces.
+
+    Parameters
+    ----------
+    height : array
+        Array of geopotential height of isentropic surfaces
+    temperature : array
+        Array of temperature on isentropic surfaces
+
+    Returns
+    -------
+    stream_func
```

Comment:
You can delete this and the blank line. Just stream_func : `pint.Quantity` is sufficient.